src/stringProcessing/relevantWords.js

```diff
@@ -215,12 +239,16 @@ function filterFunctionWords( combinations, functionWords ) {
  *
  * @param {Array} combinations The list of word combination objects.
  * @param {Function} functionWords The function containing the lists of function words.
+ * @param {string} language The language for which specific filters should be applied.
  * @returns {Array} The filtered list of word combination objects.
  */
-function filterCombinations( combinations, functionWords ) {
+function filterCombinations( combinations, functionWords, language ) {
 	combinations = filterFunctionWordsAnywhere( combinations, specialCharacters );
 	combinations = filterOneCharacterWordCombinations( combinations );
 	combinations = filterFunctionWords( combinations, functionWords );
+	if( language === "en" ) {
+		combinations = filterEndingWith( combinations, "'s", [] );
```

Comment:
This takes care of most of possessive cases, but not all. Can we also add a filter for cases like `girls'` or `Tomas'`?

Reply:
As discussed, I won't implement this at this point. Currently, all apostrophes at the end of words get trimmed at an early point, so words such as `girls'` show up as `girls` in the prominent word results. This deeper-lying issue should probably be resolved in the course of introducing wider morphological support, instead of implementing a local fix here.

Reply:
Yes, I agree. This does not look too good to my eyes. But it is outside the scope of this issue to fix it.

<img width="872" alt="screen shot 2018-06-12 at 13 58 53" src="https://user-images.githubusercontent.com/36954173/41289124-f254a1e4-6e48-11e8-821c-212fc6d5d171.png">